### DISCUSSION TITLE
Welcome to daptin Discussions!

### DISCUSSION THREAD

**Discussion by artpar** [daptin/daptin#102] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adesokanayo** [comment]
I am looking to use daptin for my new Project and also contribute to the code.  I have the following questions. 

 Is there a Discord Or slack community  ?
 is it one person maintaining this project or multiple people ?
Are new Contributors welcome ?

**Comment by artpar** [comment]
Hi @adesokanayo 

1. Here is the invite link for the slack community

https://join.slack.com/t/daptin/shared_invite/enQtMzM1NTM1NTkyMDgzLWJmZmRlN2M4YzRkOTI4MDhlNWQ1YzBiMDNhMzE0NTVmNzA3NjA5ZDdkMDExZmI0M2UyMmM2NzhlNDc3M2VhZTQ

2. This is mostly one person maintaining the project, I am also looking out for freelancers who can get involved.
3. Yes, contributors are welcome

**Reply by c-nv-s** [reply]
as not everybody is on slack, could you please consider using something like https://github.com/42wim/matterbridge to allow other people who want to contribute/participate in your community from platforms they are already using e.g. matrix, discord, telegram etc?

**Reply by artpar** [reply]
Hey @c-nv-s that's a great suggestion. Will set it up.

**Reply by c-nv-s** [reply]
is this still in the pipeline to set up? I'm only really using discord or matrix but would like to be able join any discussion if I'm going to use daptin for any serious projects.

**Reply by artpar** [reply]
@c-nv-s yes the bridge is still in pipeline, haven't been able to pick it up. All efforts are going into performance and stability changes in daptin (there have been some major stability/performance bumps). 

Don't worry you are not missing much on slack as there is hardly any activity there at this point.  Even I am not a fan of slack.

Have created a group for daptin on discord, you can join there

https://discord.gg/t564q8SQVk
